pkg/apis/configuration/validation/policy.go

```diff
@@ -284,6 +302,51 @@ func validateLogConf(logConf, logDest string, fieldPath *field.Path) field.Error
 	return allErrs
 }
 
+func validateBados(bados *v1.Bados, fieldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+
+	if bados.ApDosPolicy != "" {
+		for _, msg := range validation.IsQualifiedName(bados.ApDosPolicy) {
+			allErrs = append(allErrs, field.Invalid(fieldPath.Child("apDosPolicy"), bados.ApDosPolicy, msg))
+		}
+	}
+
+	if bados.DosSecurityLog != nil {
+		allErrs = append(allErrs, validateDosLogConf(bados.DosSecurityLog.ApDosLogConf, bados.DosSecurityLog.DosLogDest, fieldPath.Child("dosSecurityLog"))...)
+		if bados.DosSecurityLog.DosAccessLogDest != "" {
+			allErrs = append(allErrs, validateDosLogAccessLogDest(bados.DosSecurityLog.DosAccessLogDest, fieldPath.Child("dosAccessLogDest"))...)
+		}
+	}
+
+	return allErrs
+}
+
+func validateDosLogConf(logConf, logDest string, fieldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+
+	if logConf != "" {
+		for _, msg := range validation.IsQualifiedName(logConf) {
+			allErrs = append(allErrs, field.Invalid(fieldPath.Child("apDosLogConf"), logConf, msg))
+		}
+	}
+
+	err := appprotectdos.ValidateAppProtectDosLogDestination(logDest)
+	if err != nil {
+		allErrs = append(allErrs, field.Invalid(fieldPath.Child("dosLogDest"), logDest, err.Error()))
+	}
+	return allErrs
+}
+
+func validateDosLogAccessLogDest(accessLogDest string, fieldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+
+	err := appprotectdos.ValidateAppProtectDosAccessLogDest(accessLogDest)
+	if err != nil {
+		allErrs = append(allErrs, field.Invalid(fieldPath.Child("dosAccessLogDest"), accessLogDest, err.Error()))
```

Comment:
fieldPath is already dosAccessLogDest here, no need to create another child

Reply:
Done